.scripts/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Run spotlessApply using ./gradlew before pushing
+
+COMMAND="./gradlew --quiet spotlessCheck"
```

Comment:
will this only check if project is well formatted or it makes autoformatting also?

Reply:
only checks